plutus-benchmark/nofib/bench/BenchHaskell.hs

```diff
@@ -12,17 +12,17 @@ import PlutusBenchmark.NoFib.Knights qualified as Knights
 import PlutusBenchmark.NoFib.Prime qualified as Prime
 import PlutusBenchmark.NoFib.Queens qualified as Queens
 
-benchClausify :: Clausify.StaticFormula -> Benchmarkable
-benchClausify f = nf Clausify.runClausify f
+benchClausify :: String -> Clausify.StaticFormula -> Benchmark
+benchClausify name f = bench name $ whnf Clausify.runClausify f
```

Comment:
Changed `nf` to `whnf` for consistency. Note that this is just the Haskell version of the benchmarks, the CEK one is unaffected by this change.